Classes/Psmb/Newsletter/Service/FusionMailService.php

```diff
@@ -90,11 +92,8 @@ public function sendLetter($letter)
         if ($blindCarbonCopyAddress) {
             $mail->setBcc($blindCarbonCopyAddress);
         }
-        if ($format === 'plaintext') {
-            $mail->setBody($body, 'text/plain');
-        } else {
-            $mail->setBody($body, 'text/html');
-        }
+        $mail->setBody(Html2Text::convert($body), 'text/plain');
```

Comment:
Do you think it would be a able to convert rich html to plain text reasonably accurate? I need to think about this a bit more.